app/backend/graphql/project/queries/projectQueryField.ts

```diff
@@ -0,0 +1,23 @@
+import { intArg, queryField } from 'nexus'
+import { Project } from '..'
+
+export const projectQueryField = queryField('project', {
+  type: Project,
+  description: 'Returns a single project',
+  args: {
+    projectId: intArg({ description: 'Identifier for the project' }),
```

Comment:
I think we should use `idArg` to keep it consistent.

Reply:
done